Formula/w/whisperkit-cli.rb

```diff
@@ -19,6 +19,7 @@ class WhisperkitCli < Formula
   uses_from_macos "swift"
 
   def install
+    ENV["BUILD_ALL"] = "1"
```

Comment:
Why is this needed?

Reply:
It's a flag intended to enable a non-default Package.swift configuration that imports extra (all) dependencies for the macOS CLI that would normally not be included by default when imported into an iOS app.  For example https://github.com/argmaxinc/WhisperKit/blob/ce11d9b03296f2c2a912d3d3c64ddb6d6bdbdc29/Package.swift#L115-L122 - when this is merged it can use this new `BUILD_ALL` flag to import MLX. Another example of this method being used in practice https://github.com/apple/swift-nio/blob/00fedd68f4a63fd8ea58b9a7434c6ee085b9dc6e/Package.swift#L595.

Reply:
I couldn't find any usage for `BUILD_ALL` in the project, is the code not yet applied? Then it should be.